local-volume/provisioner/pkg/deleter/deleter.go

```diff
@@ -114,13 +114,11 @@ func (d *Deleter) asyncDeletePV(pv *v1.PersistentVolume, volType string, mountPa
 	}
 
 	var err error
-	switch volType {
-	case common.VolumeTypeFile:
+	switch volMode {
+	case v1.PersistentVolumeFilesystem:
 		err = d.deleteFilePV(pv, mountPath, config)
-	case common.VolumeTypeBlock:
+	case v1.PersistentVolumeBlock:
 		err = d.cleanupBlockPV(pv, mountPath, config)
-	default:
```

Comment:
Let's keep error message when the switch statement falls through to default.

Reply:
Fixed.